module-services/service-cellular/DTMFCode.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "DTMFCode.hpp"
+#include <Utils.hpp>
+#include <stdexcept>
+
+DTMFCode::DTMFCode(const char digitChar)
```

Comment:
I'm wondering - maybe add `_dtmf` user literat? :) https://en.cppreference.com/w/cpp/language/user_literal

Reply:
I think that this is more clear this way - I'd use user literal when doing e.g. units conversions etc

Reply:
I wrote because it's a conversion :) literal just state type - nothing more tbh